sku.go

```diff
@@ -163,6 +185,23 @@ func (s *SKU) GetCapabilityFloatQuantity(name string) (float64, error) {
 	return -1, &ErrCapabilityNotFound{name}
 }
 
+// GetCapabilityStringQuantity retrieves string capability with the provided name.
+// It errors if the capability is not found or the value was nil
+func (s *SKU) GetCapabilityStringQuantity(name string) (string, error) {
```

Comment:
nit - not really a quantity? `GetCapability` or `GetCapabilityString` alone perhaps?